api/services/EntranceService.js

```diff
@@ -111,6 +111,43 @@ module.exports = {
     return entranceData;
   },
 
+  // If the entrance belong to a network we send different information
```

Comment:
pourquoi laisser la question ouverte ici...

// If the entrance do not belong to a network the associated cave is populated